QR-Rangers/app/src/main/java/com/example/qr_rangers/QRCode.java

```diff
@@ -115,7 +118,7 @@ public boolean equals(Object code){
         if(!(code instanceof QRCode)){
             throw new IllegalArgumentException();
         }
-        return this.codeInfo == ((QRCode) code).codeInfo; // Possibly change to .equals() depending on what codeInfo ends up being
+        return this.codeInfo.equals(((QRCode) code).codeInfo); // Possibly change to .equals() depending on what codeInfo ends up being
```

Comment:
You can get rid of this comment
```suggestion
        return this.codeInfo.equals(((QRCode) code).codeInfo);
```